onward/app/models/MostPopularNx2.scala

```diff
@@ -30,11 +31,26 @@ case class OnwardItemNx2(
     kickerText: Option[String],
     starRating: Option[Int],
     avatarUrl: Option[String],
+    branding: Option[Branding],
 )
 
 object OnwardItemNx2 {
 
-  implicit val onwardItemWrites = Json.writes[OnwardItemNx2]
+  implicit val brandingTypeWrites = new Writes[BrandingType] {
+    def writes(bt: BrandingType) = {
+      Json.obj(
+        "name" -> bt.name,
+      )
+    }
+  }
+
+  implicit val dimentionsWrites = Json.writes[Dimensions]
```

Comment:
```suggestion
  implicit val dimensionsWrites = Json.writes[Dimensions]
```
tiny typo in the val name